src/main/java/org/sagebionetworks/web/client/widget/table/explore/TableEntityWidgetV2.java

```diff
@@ -337,31 +338,35 @@ private void setQuery(Query query, boolean isFromResults) {
 		view.setQueryWrapperPlotNavVisible(true);
 		this.view.setTableMessageVisible(false);
 		if (!isFromResults) {
-			// configure QueryWrapperPlotNav
-			OnQueryCallback onQueryChange = (newQueryJson) -> {
-				try {
-					JSONObjectAdapter adapter = ginInjector.getJSONObjectAdapter().createNew(newQueryJson);
-					this.currentQuery = new Query(adapter);
-				} catch (JSONObjectAdapterException e) {
-					ginInjector.getSynapseJSNIUtils().consoleError(e);
-				}
-			};
-			OnQueryResultBundleCallback onQueryResultBundleChange = (newQueryResultBundleJson) -> {
+			if (isShowTableOnly) {

```

Comment:
Bad diff, added new condition to support the table only:
if (isShowTableOnly) {
	view.configureTableOnly(query.getSql());
}